package-lock.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "codelab-elements",
-  "version": "4.0.0",
+  "version": "1.0.1",
```

Comment:
what is this change for? If it is not needed please remove it from PR.

Reply:
Thanks for catching that; it must have sneaked in when I pulled from the main branch